policyengine_us/tests/policy/baseline/gov/states/oh/tax/income/credits/oh_earned_income_credit.yaml

```diff
@@ -0,0 +1,26 @@
+- name: No matched credit
+  period: 2021
+  absolute_error_margin: 0.001
```

Comment:
Shouldn't need it for zeros. Also probably won't need it for others either - please try without and add if you hit a floating point issue.
```suggestion
```

Reply:
I hit the floating point issues for the matched situation, when calculation required. But the margin is not necessary for other situations.